test/test_transformers.py

```diff
@@ -688,6 +688,25 @@ def test_mha_native_args(self, nb_heads, bias):
 
             mha(query=x, key=x, value=x, key_padding_mask=pad_mask)
 
+    def test_kpm_mask_trailing_column_with_nested_tensor(self, device):
+        encoder_layer = nn.TransformerEncoderLayer(
+            d_model=256,
+            nhead=4,
+            dim_feedforward=512,
+            activation='gelu',
+            norm_first=False,
+            batch_first=False,
+        )
+        transformer_encoder = nn.TransformerEncoder(encoder_layer, num_layers=3, enable_nested_tensor=True).to(device)
```

Comment:
Is the output of

```
transformer_encoder_with_nt = nn.TransformerEncoder(encoder_layer, num_layers=3, enable_nested_tensor=True).to(device)
output0 = transformer_encoder(src=x, src_key_padding_mask=mask)
```

going to be the same as without nt?

```
transformer_encoder_without_nt = nn.TransformerEncoder(encoder_layer, num_layers=3, enable_nested_tensor=False).to(device)
output1 = transformer_encoder(src=x, src_key_padding_mask=mask)
```

Are `output0` and `output1` going to produce the same output?